doc/ReleaseNotes/_ReleaseNotes.md

```diff
@@ -38,9 +38,11 @@
 * Add support for the `CollectionViewSource.ItemsPath` property
 * Fixed support for dots in resource names (#700)
 * Add support for `BindingExpression.UpdateSource()`
+* Updated Android version to target Android 9.0
 
 ### Breaking changes
 * Make `UIElement.IsPointerPressed` and `IsPointerOver` internal
+* Removed Target for Android 8.0. You will now need your projects to target Android 9.0 to avoid errors.
```

Comment:
Remove this line

Reply:
Rewrote the line to better reflect the changes